{{cookiecutter.project_slug}}/requirements.txt

```diff
@@ -0,0 +1 @@
+-r requirements/heroku.in
```

Comment:
This should include the pinned version of the requirements file, aka `requirements/heroku.txt` (it doesn't exist but will be created by Drifter).

Reply:
done